src/Components/Navbar/Navbar.jsx

```diff
@@ -61,7 +65,7 @@ function Navbar() {
                             <div className="nav-list--item__icon--wrapper">
                                 <span>
                                         <i className="fas icon fa-heart">
-                                            <span className="status-badge notification-badge">6</span>
+                                            <span className="status-badge notification-badge">{isAuthorized ? wishListData.length : 0}</span>
```

Comment:
Suggestion: You can choose not to show any badge if the user is not logged in. Not even 0.